bundle.proto

```diff
@@ -27,14 +27,35 @@ message Accepted {
   string validator_identity = 2;
 }
 
+message ForwardedToValidator {
+  // Slot at which bundle was forwarded.
+  uint64 slot = 1;
+
+  // Validator identity bundle was forwarded to.
+  string validator_identity = 2;
+}
+
 // Indicates the bundle was dropped and therefore not forwarded to any validator.
 message Rejected {
   oneof reason {
+    // The bundle did not win the auction it was being considered in.
     StateAuctionBidRejected state_auction_bid_rejected = 1;
-    WinningBatchBidRejected winning_batch_bid_rejected = 2;
+
+    // Deprecated, this field is no longer set by the Block-Engine.
+    WinningBatchBidRejected winning_batch_bid_rejected = 2 [deprecated = true];
+
+    // Bundle failed simulation.
     SimulationFailure simulation_failure = 3;
+
+    // Some internal error occurred with the Block-Engine.
     InternalError internal_error = 4;
-    DroppedBundle dropped_bundle = 5;
+
+    // Indicates accepted by the block-engine and forwarded to a jito-solana validator.
+    // Deprecated, start using `other`.
+    DroppedBundle dropped_bundle = 5 [deprecated = true];
+
+    // Set to really high number so we can potentially add other fields before.
+    string other = 100;
```

Comment:
can remove this. seems useless at first glance and will be another match statement in code for people reading this message